decidim-home/lib/decidim/home.rb

```diff
@@ -1,5 +1,4 @@
 require "decidim/home/engine"
-
```

Comment:
I find many changes of this kind (removal of empty lines). Do you do them on purpose? If not, better keep the code as is if we're not improving readability.